pyxem/signals/virtual_dark_field_image.py

```diff
@@ -150,12 +150,12 @@ def get_vdf_segments(
         n = vdfsegs.segments.axes_manager.navigation_axes[0]
         n.name = "n"
         n.units = "number"
-        vdfsegs.vectors_of_segments.axes_manager.set_signal_dimension(1)
-        vdfsegs.vectors_of_segments = transfer_signal_axes(
-            vdfsegs.vectors_of_segments, self.vectors
-        )
-        n = vdfsegs.vectors_of_segments.axes_manager.navigation_axes[0]
-        n.name = "n"
-        n.units = "number"
+        #vdfsegs.vectors_of_segments.axes_manager.set_signal_dimension(1)
+        #vdfsegs.vectors_of_segments = transfer_signal_axes(
+        #    vdfsegs.vectors_of_segments, self.vectors
+        #)
+        #n = vdfsegs.vectors_of_segments.axes_manager.navigation_axes[0]
+        #n.name = "n"
+        #n.units = "number"
```

Comment:
@pc494 
Not sure exactly what this code is supposed to do?  This causes a failure because there aren't enough dimensions to do what is being asked of it.  It seems like a bug and with the new stricter dimension checking in hyperspy I don't think that it is necessary but I am not entirely sure.

Reply:
I suggest you delete the commented code and raise a warning to the effect of "Changed in 1.0.0 use with caution".